### PR TITLE
create Userprofile

### PR DESCRIPTION
**Pull request by ZJW-92**
create user account including name, email, salary per hour and total hours.

### CODE REVIEW

File: src/main/java/org/coffeemine/app/spring/UserProfile.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package org.coffeemine.app.spring;
+
+import com.vaadin.flow.router.Route;
+
+@Route
+public class UserProfile extends View {
+
+    UserProfile() {
+    super();
+    add(new profileView());
```
Comment (RuthgerD) [inline src/main/java/org/coffeemine/app/spring/UserProfile.java]: This file is unneeded, profile view is separate and should not be part of the View.
Instead profileView should be accessed via the account button in the future, but this is not required by the current issue.

File: src/main/java/org/coffeemine/app/spring/UserProfileComponent.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+import com.vaadin.flow.component.icon.VaadinIcon;
+import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+import com.vaadin.flow.component.tabs.Tab;
+import com.vaadin.flow.component.Text;
+import com.vaadin.flow.component.textfield.TextField;
+import com.vaadin.flow.component.formlayout.FormLayout;
+
+
+class profileView extends VerticalLayout {
```
Comment (RuthgerD) [inline src/main/java/org/coffeemine/app/spring/UserProfileComponent.java]: Please name your class according to the file name. (and according to java standards e.g. has to start with capital letter) -> ProfileView.java as file name and ProfileView als class name.

File: src/main/java/org/coffeemine/app/spring/UserProfileComponent.java [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        details.add(accountDetails);
+
+        VerticalLayout states = new VerticalLayout();
+        states.setWidthFull();
+        states.add(new H3("Account states"));
+
+        FormLayout accountStates = new FormLayout();
+        TextField salaryPerHourFiled = new TextField();
+        salaryPerHourFiled.setPlaceholder("Salary per hour");
+        accountStates.addFormItem(salaryPerHourFiled, "Salary per hour:");
```
Comment (RuthgerD) [inline src/main/java/org/coffeemine/app/spring/UserProfileComponent.java]: These arent supposed to be TextFields, use Span instead.

File: src/main/java/org/coffeemine/app/spring/UserProfileComponent.java [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+import com.vaadin.flow.component.html.Span;
+import com.vaadin.flow.component.html.H1;
+import com.vaadin.flow.component.html.H2;
+import com.vaadin.flow.component.html.H3;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.icon.Icon;
+import com.vaadin.flow.component.icon.VaadinIcon;
+import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+import com.vaadin.flow.component.tabs.Tab;
```
Comment (RuthgerD) [inline src/main/java/org/coffeemine/app/spring/UserProfileComponent.java]: Please clean out any unused libraries (Span, H1, Div, Tab, Text)

File: src/main/java/org/coffeemine/app/spring/UserProfileComponent.java [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+        logOut.addThemeVariants(ButtonVariant.MATERIAL_CONTAINED);
+        add(logOut);
+
+    }
+}
+
+
+
+
+
```
Comment (RuthgerD) [inline src/main/java/org/coffeemine/app/spring/UserProfileComponent.java]: Just nagging here but please remove these new lines.

File: src/main/java/org/coffeemine/app/spring/UserProfile.java (hunk 1 above)
Comment (ZJW-92) [reply src/main/java/org/coffeemine/app/spring/UserProfile.java]: ah, I will remove this

File: src/main/java/org/coffeemine/app/spring/UserProfileComponent.java (hunk 2 above)
Comment (ZJW-92) [reply src/main/java/org/coffeemine/app/spring/UserProfileComponent.java]: okay, will change to capital letter.

File: src/main/java/org/coffeemine/app/spring/UserProfileComponent.java (hunk 3 above)
Comment (ZJW-92) [reply src/main/java/org/coffeemine/app/spring/UserProfileComponent.java]: fix it already

File: src/main/java/org/coffeemine/app/spring/UserProfileComponent.java (hunk 4 above)
Comment (ZJW-92) [reply src/main/java/org/coffeemine/app/spring/UserProfileComponent.java]: okay ！

File: src/main/java/org/coffeemine/app/spring/UserProfileComponent.java (hunk 5 above)
Comment (ZJW-92) [reply src/main/java/org/coffeemine/app/spring/UserProfileComponent.java]: okay！

File: src/main/java/org/coffeemine/app/spring/UserProfile.java (hunk 1 above)
Comment (RuthgerD) [reply src/main/java/org/coffeemine/app/spring/UserProfile.java]: Please remove the old file :)

**Pull request merged**